plugins/BEdita/Core/tests/TestCase/Model/Action/SignupUserActionTest.php

```diff
@@ -165,7 +166,7 @@ public function executeProvider()
     public function testExecute($expected, array $data)
     {
         if ($expected instanceof \Exception) {
-            $this->expectException(get_class($expected));
+            static::expectException(get_class($expected));
```

Comment:
[minor minor] `expectException` is not a static method.

Reply:
ok, we better change all of them in one shot